pywebdriver/plugins/telium_driver.py

```diff
@@ -64,6 +64,9 @@ def get_payment_info_from_price(self, price, payment_mode):
 def payment_terminal_transaction_start():
     app.logger.debug("Telium: Call payment_terminal_transaction_start")
     payment_info = request.json["params"]["payment_info"]
+    if payment_info.get("refund") or payment_info["amount"] < 0:
```

Comment:
This does not look related to the rest of the PR

Reply:
It's related to the PR [785 of pos](https://github.com/OCA/pos/pull/785) and shifts the check from the pos to the terminal. Before the pos was raising an error if the amount was <=0. We can exclude it here but the customer requested the feature to allow refunds.

Reply:
Ok I understand (the PR https://github.com/OCA/pos/pull/785/files is closed FYI)